models/DataObject/Classificationstore/CollectionGroupRelation.php

```diff
@@ -60,6 +60,24 @@ public static function create()
         return $config;
     }
 
+    /**
+     * @param int|null $colId
+     * @param int|null $groupId
+     *
+     * @return self|null
+     */
+    public static function getByColAndGroupId($colId = null, $groupId = null)
```

Comment:
how about we change the method name to `getByGroupAndColId` along with correct param order? so that it is consistent with `KeyGroupRelation::getByGroupAndKeyId`

Reply:
Done